dls-controls/malcolmjs
# Feature/theme shenanigans

## Description

Added simple theme editor + ability to set theme via query parameters

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- npm start
- navigate to localhost:3000
- check theme is as per default
- click the cog in the nav bar
- try editing the theme
- navigate to localhost:3000/gui/?type=light&primary=red
- check the theme is set accordingly
